### ISSUE TITLE
Großes Widget mit mehreren Orten?

### ISSUE THREAD

**Issue by torretx** [marcusraitner/COVID-19-Dashboard#20]
Wäre es möglich mehrere Landkreise / Städte in einem großen Widget anzeigen zu lassen?

**Comment by marcusraitner** [comment]
Machbar ist es auf jeden Fall. Aber was wäre der Vorteil gegenüber der Variante einfach für jeden Ort ein eigenes Widget auf dem Homescreen zu haben. Geht es nur um den Platz?

**Comment by JDTm** [comment]
Ich habe es so gelöst: meinen Heimatort und mein Wohnort angelegt und dann als Stapel (das eine Widget auf das andere gezogen). Per Wischgeste beide zu tauschen finde ich super.

**Comment by torretx** [comment]
Stimmt, daran habe ich nicht gedacht. Habe nur an die Möglichkeiten des anderen bekannten Corona Widgets gedacht j d dort hatte ich eben zwei Orte sofort auf einen Blick ohne viel Platz zu verlieren.

**Comment by marcusraitner** [comment]
> Ich habe es so gelöst: meinen Heimatort und mein Wohnort angelegt und dann als Stapel (das eine Widget auf das andere gezogen). Per Wischgeste beide zu tauschen finde ich super.

Das ist eine sehr gute und platzsparende Idee. Daran hatte ich noch gar nicht gedacht.

**Comment by marcusraitner** [comment]
Ich lasse das trotzdem noch offen hier, weil ich mir ohnehin anschauen wollte, wie das mit den verschiedenen Widgetgrößen funktioniert.
